test/drenv/__main__.py

```
@@ -292,6 +294,24 @@ def configure_logging(args):
     )
 
 
+def _git_version():
+    """
+    Return the current git commit hash with a '-dirty' suffix if the working
+    tree has uncommitted changes. Falls back to 'unknown' if git is unavailable.
+    """
+    try:
+        result = subprocess.run(
+            ["git", "describe", "--always", "--dirty", "--no-tags"],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)